### ISSUE TITLE
Bug report - cannot import name 'get_number_ts' 

### ISSUE THREAD

**Issue by rhysdg** [snakers4/silero-vad#470]
## 🐛 Bug

Hey there! Thanks for all the great work! Loving working with the onnx deployment right now. Just a heads up that the latest commit in the last hour below breaks torch hub with:

```
ImportError: cannot import name 'get_number_ts' from 'utils_vad' (/root/.cache/torch/hub/snakers4_silero-vad_master/utils_vad.py)

```

https://github.com/snakers4/silero-vad/commit/74c3f7f3fb8c06b6ee2e0efa7d8cbcb8cf421362

## To Reproduce

```
model, utils = torch.hub.load(repo_or_dir='snakers4/silero-vad',
                              model='silero_vad',
                              force_reload=True,
                              onnx=USE_ONNX)
```

## Environment

I've tried this with Ubuntu 22.04, colab, Jetpack etc Let me know if there's anything I can do!

**Comment by snakers4** [comment]
Hi, removed old utils related to the models deprecated a long time ago, but forgot to remove their references from the hubconf file.
Please check now, is it fixed?
https://github.com/snakers4/silero-vad/commit/06b9e17c1e12284fc59643899de356fc1398e031

**Comment by rhysdg** [comment]
> Hi, removed old utils related to the models deprecated a long time ago, but forgot to remove their references from the hubconf file. Please check now, is it fixed? [06b9e17](https://github.com/snakers4/silero-vad/commit/06b9e17c1e12284fc59643899de356fc1398e031)

Perfect! Thanks for the quick response and looking forward to working with the new version

**Comment by snakers4** [comment]
Ok
